src/main/java/com/bullhornsdk/data/model/entity/core/paybill/JobCode.java

```diff
@@ -0,0 +1,132 @@
+package com.bullhornsdk.data.model.entity.core.paybill;
+
+import com.bullhornsdk.data.api.helper.json.DynamicNullValueFilter;
+import com.bullhornsdk.data.model.entity.core.standard.BusinessSector;
+import com.bullhornsdk.data.model.entity.core.standard.Category;
+import com.bullhornsdk.data.model.entity.core.standard.CorporateUser;
+import com.bullhornsdk.data.model.entity.core.standard.CorporationDepartment;
+import com.bullhornsdk.data.model.entity.core.type.*;
+import com.bullhornsdk.data.model.entity.embedded.OneToMany;
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.Getter;
+import org.joda.time.DateTime;
+
+import java.math.BigDecimal;
+
+@Data
+@JsonFilter(DynamicNullValueFilter.FILTER_NAME)
+public class JobCode extends AbstractEntity implements QueryEntity, UpdateEntity, CreateEntity, EditHistoryEntity, DateLastModifiedEntity, SoftDeleteEntity {
+    Integer id;
+    OneToMany<BusinessSector> businessSectors;
+    OneToMany<Category> categories;
+    DateTime customDate1;
+    DateTime customDate2;
+    DateTime customDate3;
+    BigDecimal customFloat1;
+    BigDecimal customFloat2;
+    BigDecimal customFloat3;
+    Integer customInt1;
+    Integer customInt2;
+    Integer customInt3;
+    @JsonIgnore
+    @Getter(onMethod = @__(@JsonProperty))
```

Comment:
Additionally, I don't even think we need to specify the `@Getter(...)` at all anymore. Since `customText1` will match what is returned in the JSON of a response

Reply:
> From JDK 8 on, the standard for this is in the format: `@Getter(onMethod_ = {@JsonProperty})`

Source of this? 

> Additionally, I don't even think we need to specify the `@Getter(...)` at all anymore. Since `customText1` will match what is returned in the JSON of a response

I need to specify the getter explicitly to apply the JsonProperty tag to the getter method, thus allowing the field to be serialized.

Reply:
> Source of this?

@scriptom Source: https://projectlombok.org/api/lombok/Getter

> I need to specify the getter explicitly to apply the JsonProperty tag to the getter method, thus allowing the field to be serialized.

We'll talk over slack about this portion

Reply:
Okay. This makes sense now. Resolved thread